### ISSUE TITLE
Scenesfrancophones merge into single artifact

### ISSUE THREAD

**Issue by saumier** [culturecreates/artsdata-planet-scenesfrancophones#18]
As a consumer of data from Artsdata, I don't have any need for 3 different artifacts for scenesfrancophones, and there are problems with the data because there are several PostalAddresses and GeoCoordinates for the same place because they use blank nodes which accumulate in each graph and look like different PostalAddresses even though they are not.  

To solve these data issues, one idea is to merge the 3 API calls for events, artists and places just before pushing to artsdata, and push only a single artifact. Note: the merge with RDF is very simple and only needs to have the 3 files concatenated into a list and saved in a single file.

Please merge the 3 data files just before pushing to Artsdata and call the new artifact `scenesfrancophones-data`.  There should be a single workflow to run (it can call other workflows). You can keep the individual dump commits in github as a steps, but a merged dump will need to be added.

If this approach works then we can delete the older artifacts scenesfrancophones-places, scenesfrancophones-artists, scenesfrancophones-events.

This is high priority because it is needed to complete the scenesfrancophones events for import into CMS calendar artsCultureNB.

**Comment by dev-aravind** [comment]
@saumier There is now a single workflow that fetches data from scenesfrancophones. Please review this and let me know if you need any changes.

**Comment by saumier** [comment]
@dev-aravind This looks good. I deleted the older 3 artifacts and we will keep this one only going forward.
The SHACL errors for the startDate were because the datatype was schema:Date instead of xsd:dateTime. I have a general transformation in Artsdata platform that normally fixes this across the board but it was not being included in the pipeline. I added it so now the datatype is fixed for all incoming data.
